apps/scope/pkg/apis/scope/v0alpha1/zz_generated.openapi_violation_exceptions.list

```diff
@@ -0,0 +1,11 @@
+API rule violation: list_type_missing,github.com/grafana/grafana/apps/scope/pkg/apis/scope/v0alpha1,FindScopeDashboardBindingsResults,Items
```

Comment:
Should we care about these violations?

Reply:
We can have a look at them after but they are not important